src/resultset/formatters/resultset_replyverbose.c

```diff
@@ -178,6 +182,15 @@ static void _ResultSet_VerboseReplyWithMap(RedisModuleCtx *ctx, SIValue map) {
 	rm_free(str);
 }
 
+static void _ResultSet_VerboseReplyWithPoint(RedisModuleCtx *ctx, SIValue point) {
+	// point({latitude:56.7, longitude:12.78})
+	char buffer[256];
+	int bytes_written = sprintf(buffer, "point({latitude:%f, longitude:%f})",
```

Comment:
Neo4j prints points using the format `{x: lat_val, y: lon_val}`. Would you prefer to be consistent here?